backend/app/schemas/user.py

```diff
@@ -0,0 +1,62 @@
+from enum import Enum
```

Comment:
Let's add a docstring describing what these schemas do
```
"""
User validation schemas etc. etc.
"""
```
and short one line description comments for these classes where possible.